lib/osf-components/addon/components/file-actions-menu/styles.scss

```diff
@@ -22,3 +22,8 @@
     composes: FakeLink from '../button/styles.scss';
     margin: 6px 0;
 }
+
+.RenameLink {
```

Comment:
Do we need this class?

Reply:
When we use the DropdownItem class, it places it a few pixels higher than would be ideal. Overriding it with this class gives it a bit better handling than the below with only DropdownItem:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/82047646/170561876-fe3e9a57-ceb5-4dab-9b72-d39752b2d18f.png">

Reply:
This has now also been removed by using the DropdownItem class and converting the OsfLink to a Button.